+nix/Block.m

```diff
@@ -29,10 +29,14 @@
             obj@nix.Entity(h);
             obj.info = nix_mx('Block::describe', obj.nix_handle);
             
-            obj.dataArraysCache = {};
-            obj.sourcesCache = {};
-            obj.tagsCache = {};
-            obj.multiTagsCache = {};
+            obj.dataArraysCache.lastUpdate = 0;
+            obj.dataArraysCache.data = {};
+            obj.sourcesCache.lastUpdate = 0;
+            obj.sourcesCache.data = {};
+            obj.tagsCache.lastUpdate = 0;
+            obj.tagsCache.data = {};
+            obj.multiTagsCache.lastUpdate = 0;
+            obj.multiTagsCache.data = {};
```

Comment:
In the future - we could maybe introduce smth like a ProxyList class to hold (cache) child objects